op-acceptance-tests/tests/base_multi/advance_multiple.go

```diff
@@ -0,0 +1,45 @@
+package base_multi
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCLAdvanceMultiple verifies two L2 chains advance when using separate CLs
+func TestCLAdvanceMultiple(gt *testing.T) {
+	t := devtest.ParallelT(gt)
```

Comment:
nit: this is the only test in this package, so whether it's Parallel or Serial makes no difference. As such, should we default to the, safer, Serial? When more tests are introduced it may impact them. Or maybe it's safe and this comment is irrelevant.